src/display.rs

```diff
@@ -281,13 +281,15 @@ fn get_visible_width(input: &str) -> usize {
     // Search for every instance of "\e[" indicating the start of a color code
     // then count the number of charactere between the string and the first 'm'
     // charactere indicating the end of the code color.
-    for (idx, _) in input.match_indices("\u{1b}[") {
-        for c in input.chars().skip(idx) {
+    let matches: Vec<_> = input.match_indices("\u{1b}[").collect();
+    if matches.len() > 0 {
+        for c in input.chars().skip(matches[0].0) {
             nb_invisible_char += 1;
             if c == 'm' {
                 break;
             }
         }
+        nb_invisible_char += 2;
```

Comment:
This `+= 2` intrigues me... Where it come from? It represent neither the `\t{1b}[` nor the `m`